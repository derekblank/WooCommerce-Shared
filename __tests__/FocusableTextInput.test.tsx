import { render } from '@testing-library/react-native';
import FocusableTextInput from '../UI/FocusableTextInput';

test('renders FocusableTextInput', () => {
  const { toJSON } = render(<FocusableTextInput />);
  expect(toJSON()).toMatchSnapshot();
});