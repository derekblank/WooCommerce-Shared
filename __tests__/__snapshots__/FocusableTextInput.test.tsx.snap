// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FocusableTextInput 1`] = `
<TextInput
  onBlur={[Function]}
  onFocus={[Function]}
  style={
    [
      {
        "borderRadius": 4,
        "borderWidth": 1,
        "paddingHorizontal": 10,
        "paddingVertical": 5,
      },
      {
        "borderColor": "gray",
      },
      undefined,
    ]
  }
/>
`;
